### PR TITLE
add lecture 1 notes

### PR DESCRIPTION
**Pull request by ngwattcos**

### CODE REVIEW

File: versioned_docs/version-2022sp/lecture1.md [hunk 1]
```diff
@@ -3,4 +3,532 @@ id: lecture1
 title: Lecture 1
 ---
 
-To be added
+Lecture Slides(to be added later)
```
Comment (Enochen) [inline versioned_docs/version-2022sp/lecture1.md:6]: Can we add lecture slides right now?

File: versioned_docs/version-2022sp/lecture1.md [hunk 2]
```diff
@@ -3,4 +3,532 @@ id: lecture1
...
+
+### Basic JavaScript Syntax
+
+#### Variables
+
+There are three ways to create variables in JS:
+
+1. ~~`var x = 5`~~
+2. `let x = 5`
+3. **`const x = 5`**
```
Comment (Enochen) [inline versioned_docs/version-2022sp/lecture1.md:33]: Can we clarify on what const actually does? I think immutability is the wrong word, it just prevents reassignment.

**Review by Enochen — CHANGES_REQUESTED** [review]
Reminder to add demo code

File: versioned_docs/version-2022sp/lecture1.md (hunk 1 above)
Comment (ngwattcos) [reply versioned_docs/version-2022sp/lecture1.md]: added as a link

File: versioned_docs/version-2022sp/lecture1.md (hunk 2 above)
Comment (ngwattcos) [reply versioned_docs/version-2022sp/lecture1.md:33]: Changed to mean non-reassignable based on our language today in class

File: versioned_docs/version-2022sp/lecture1.md [hunk 3]
```diff
@@ -3,4 +3,532 @@ id: lecture1
 title: Lecture 1
 ---
 
-To be added
+[Lecture Slides](https://docs.google.com/presentation/d/14LCzgpdo6381cS3gfda4fxdumJyl0WS2tRW5PY4EqGg/edit?usp=sharing)
```
Comment (Enochen) [inline versioned_docs/version-2022sp/lecture1.md:6]: That's the private folder, I made a public version here: https://docs.google.com/presentation/d/1NX7v1mRL9sih1P6lujQnDVvjNYPcDgiE_XXabcLy6fc/edit?usp=sharing

**Review by Enochen — APPROVED** [review]
Looks good other than the slides link

**Pull request merged**